canonical-websites/www.ubuntu.com
# Remove vanilla framework 0.7.7 and uvt 0.0.35

## Done

* deleted the old frameworks
* deleted the old _v1_section folder

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- See that the site looks ok
